### PR TITLE
Feature/#m mb63qqa import csv

### PR DESCRIPTION
**Pull request by ZrodoZrodo**

### CODE REVIEW

File: components/common/Button.tsx [hunk 1]
```diff
@@ -3,13 +3,15 @@ children: string,
...
+export const Button = ({ children,type,color,onClick,disabled }: ButtonProps ) => {
   return (
     <button
       type={type?type:"button"}
-      className={`${color?color:"bg-primary-red"} text-light-primary-text py-2 px-4`}
+      className={`${color?color:disabled?"bg-secondary-background":"bg-primary-red"} text-light-primary-text py-2 px-4`}
       onClick={onClick}
+      disabled={disabled}
     >
       {children}
```
Comment (MichalTomczakSE) [inline components/common/Button.tsx:18]: Maybe change children prop to be "ReactNode" instead of string?

File: components/common/Header.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ const links = [
 
 export const Header = () => {
     return (
-        <header className="bg-navbar-background">
+        <header className="bg-navbar-background ">
```
Comment (MichalTomczakSE) [inline components/common/Header.tsx]: There is no changes, delete this in next commit.

File: components/common/loaderCSV/loaderCSV.tsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
+import {ChangeEvent, useEffect, useState} from "react";
+import {ValidationCSV} from "@/components/common/loaderCSV/validationCSV";
+import {Button} from "@/components/common/Button";
+
+interface csv{
```
Comment (MichalTomczakSE) [inline components/common/loaderCSV/loaderCSV.tsx]: change name to CsvFileProps.

File: components/common/loaderCSV/loaderCSV.tsx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+                        <td scope="col" className="px-6 py-3">
+                            {errorRows&&Array.from(errorRows).join(',')}
+                            {headerValid?"":"0"}
+                        </td>
+                    </tr>
+                    </tbody>
+                </table>
+            </div>
+            <div className={"space-y-3 space-x-2"}>
+            <Button disabled={errorRows?.size===0||array.length===0} onClick={deleteErrors}>Usuń błędne rekordy</Button><Button disabled={errorRows?.size!==0||array.length===0} onClick={submitCsv}>Wyślij do bazy </Button>
```
Comment (MichalTomczakSE) [inline components/common/loaderCSV/loaderCSV.tsx]: Add some line spacing before attributes for better code view.

File: components/common/loaderCSV/validationCSV.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import {object} from "prop-types";
+
+
+interface csv{
```
Comment (MichalTomczakSE) [inline components/common/loaderCSV/validationCSV.ts]: Same as above, change name. If you are using same interface in two files, create separate type file, and export/import it.

File: pages/index.tsx [hunk 6]
```diff
@@ -1,10 +1,13 @@
 import {Header} from "@/components/common/Header";
+import {LoaderCSV} from "@/components/common/loaderCSV/loaderCSV";
 
 export default function Home() {
     return (
-        <>
```
Comment (MichalTomczakSE) [inline pages/index.tsx:5]: Before merge, you can delete it and leave HomePage empty.

File: yarn.lock [hunk 7]
```diff
@@ -983,6 +983,22 @@
     "@typescript-eslint/types" "5.59.0"
```
Comment (MichalTomczakSE) [inline yarn.lock]: Delete yarn.lock, and commit package.json instead.

**Review by MichalTomczakSE — APPROVED** [review]

**Review by AgnieszkaPawlak — APPROVED** [review]

**Pull request merged**